docker-local/docker-compose.yml

```diff
@@ -59,6 +60,16 @@ services:
       - storjsim
       - authservice
       - tracing
+  linksharing:
+    image: golang:1.17
+    entrypoint: /scripts/start-linksharing.sh
+    volumes:
+      - localgopath:/go/storjpath
+      - ${GATEWAY_SRC_DIR}:/code
+      - ./start-linksharing.sh:/scripts/start-linksharing.sh
+      - storjsimconfig:/config
```

Comment:
We probably should include a ENV to reference a geoip DB from the local machine here.

Reply:
It appears it works without. 

Maybe we can mock a file which puts the Docker subnet (172.*) in Atlanta.